.github/workflows/cargo-deny.yml

```diff
@@ -0,0 +1,54 @@
+name: cargo deny checks
+
+on:
+  workflow_call:
+    inputs:
+      build-tools-image:
+        required: false
+        type: string
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  cargo-deny:
+    strategy:
+      matrix:
+        ref: >-
+          ${{
+            fromJSON(
+              github.event_name == 'schedule'
+                && '["main","release","release-proxy","release-compute"]'
+                || format('["{0}"]', github.sha)
+            )
+          }}
+
+    runs-on: [self-hosted, small]
+
+    container:
+      image: ${{ inputs.build-tools-image || 'neondatabase/build-tools:pinned' }}
+      credentials:
+        username: ${{ secrets.NEON_DOCKERHUB_USERNAME }}
+        password: ${{ secrets.NEON_DOCKERHUB_PASSWORD }}
+      options: --init
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ matrix.ref }}
+
+      - name: Check rust licenses/bans/advisories/sources
+        run: cargo deny check --hide-inclusion-graph ${{ github.event_name == 'schedule' && 'advisories' || 'all' }}
```

Comment:
Let's move `${{ github.event_name == 'schedule' && 'advisories' || 'all' }}` to `env:` for the step

```suggestion
        run: cargo deny check --hide-inclusion-graph ${{ github.event_name == 'schedule' && 'advisories' || 'all' }}
```